### DISCUSSION TITLE
Make the video window draggable by clicking anywhere inside it instead of clicking on just the title bar

### DISCUSSION THREAD

**Discussion by Psybertr0n** [f1multiviewer/issue-tracker#92] [Feature requests] [answered]
As it is currently known, the title bar on videos is a bit glitchy and it doesn't stay visible on mouse hover. Consequently dragging video windows around can sometimes get annoying. Would it be possible to make the video window draggable by clicking anywhere inside it? A simple left click within the video window is not bound to any action right now so I can imagine it could behave this way:
- single click inside video does nothing
- single click and hold inside video can let you drag the window
- double click to go full screen
- right click to go into menu

This would be in line with how mpv Player behaves. You can keep the title bar and reveal it only on mouse hover but don't require that we click on that bar alone if we want to drag the window around.

**Comment by f1multiviewer** [comment, marked as answer]
Fixed in the next release!

**Reply by dominicusmento** [reply]
Now it's draggable from anywhere but stopped working with Microsoft PowerToys which allows quick snap and resize to defined monitor positions.. It seems that now OS doesn't register it as a regular 'app window' as also snap to the monitor edge doesn't make quick positioning like with all the other apps..
Any way of making this an option to turn it on/off per preference ?
(it was working fine in v1.4.3)

Edit: downgraded to v1.4.3 - works again..
(also, title bar is not so glitchy if you get how to use it.. 
U should place a mouse on the title bar placeholder and wait for it to show up without moving the mouse.. then click and hold for dragging)

**Comment by JJWatMyself** [comment]
This is working on MS Windows now in 1.4.4 but has the following impacts:
 - clicking anywhere on the window will drag it
 - volume slider can no longer be dragged
 - change video resolution requires an addition click to close.

**Reply by spambas** [reply]
volume slider was giving me issues too
